packages/design/sass/styles.scss

```diff
@@ -166,3 +166,60 @@ iframe:focus {
   padding-bottom: 62.5%;
   height: 0;
 }
+
+.usa-fieldset {
+  width: 100%;
+
+  .usa-legend,
+  .usa-alert__text {
+    font-size: 1rem;
+  }
+
+  .usa-legend {    
+    font-weight: normal;
+    line-height: 1.5rem;
+    text-transform: uppercase;
+  }
+
+  h3.section-heading-h3 {
+    font-size: 1.375rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+  }
+
+  .usa-alert__text {
+    span {
+      display: inline-block;
+      vertical-align: top;
+      margin-right: 1rem;
+    }
+  }
+}
+
+.edit-component-panel {
+  input,
+  select {
+    background: #f4f1f9;
+    font-weight: 700;
+  }
+
+  .usa-fieldset {
+    margin-top: 0;
+  }
+}
```

Comment:
Same here, there are color theme tokens and corresponding utilities (eg, `bg-primary-light`), text utilities (`text-bold`), and margin (`margin-top-0`).

Reply:
The utility class for colors are not specific enough for certain backgrounds, such as the light violet that's in the focused input field when in edit mode. When I attempted to use color: color('violet-5'), it wasn't recognized. I was forced to report to the hex code.